specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -113,16 +113,16 @@ export function IntegratedRecordSelector({
 
   const [isDialogOpen, handleOpenDialog, handleCloseDialog] = useBooleanState();
 
-  const isTaxonTreeDefItemTable =
-    collection.table.specifyTable.name === 'TaxonTreeDefItem';
-
   const isAttachmentTable =
     collection.table.specifyTable.name.includes('Attachment');
 
   const isCOJO =
     relationship.relatedTable.name === 'CollectionObjectGroupJoin' &&
     relationship.name === 'children';
 
+  const isTaxonTreeDefItemTable =
+    collection.table.specifyTable.name === 'TaxonTreeDefItem';
+
```

Comment:
This will disable adding ranks through the taxontreedefitem subview all across Specify and not just in the tree creation dialog. Is there any place in Specify where we still want the ability to add a rank through the subview?

If so, a potential alternative could be to change the treedef form definition to not show the ranks in the form.